### PR TITLE
Haproxy consolidate apps

### PR DESCRIPTION
**Pull request by obino**
There are 2 methods to effectively do the same thing: add an appserver/server to an haproxy service. This PR consolidates them to 1.

### CODE REVIEW

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/4613/

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/4616/

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/4626/

File: AppController/lib/haproxy.rb [hunk 1]
```diff
@@ -409,7 +383,7 @@ def self.initialize_config(connect_timeout)
     # parameters for both haproxies.
     File.open(BASE_CONFIG_FILE, 'w+') { |dest_file| dest_file.write(base_config) }
     File.open(SERVICES_BASE_FILE, 'w+') { |dest_file|
-      dest_file.write(base_config.sub('/stats', '/service-stats'))
+      dest_file.write(base_config.sub('/stats', '/services-stats'))
```
Comment (cdonati) [inline AppController/lib/haproxy.rb]: I would prefer `service-stats` since "services stats" is grammatically incorrect.

Comment (obino) [reply AppController/lib/haproxy.rb]: LOL thanks, you got it.

**Comment by cdonati** [comment]
@obino Looks like there might be an issue with your latest commit: https://ocd.appscale.com:8080/job/Daily%20Build/4663/

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/4666/

**Comment by obino** [comment]
@cdonati yep thanks for pointing out: started a new build.

**Pull request merged**